sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/EntityKey.java

```diff
@@ -21,7 +21,7 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-@Target({ElementType.METHOD, ElementType.PARAMETER, ElementType.TYPE})
```

Comment:
I remove the option to have it on Methods and Params for now. I will open an issue where we can discuss it in detail.